### ISSUE TITLE
All layers with normalization not updating

### ISSUE THREAD

**Issue by mshubhankar** [awslabs/fast-differential-privacy#40]
I get a bunch of layers that do not update and print the error:
>  [..] are not supported by privacy engine; these parameters are not requiring gradient nor updated.

These layers have normalization. I tried both GroupNorm and LayerNorm, and they have the same issue.
The issue disappears if normalization is substituted with `nn.Identity() `.

Here is a code snippet of how the normalization is defined:
Please note that they are slightly customized by inheriting from `nn.GroupNorm()` and `nn.LayerNorm()` respectively.
```
class GroupNorm32(nn.GroupNorm):
    def forward(self, x):
        return super().forward(x.float()).type(x.dtype)

class LayerNorm2D(nn.LayerNorm):
    def forward(self, x):
        if len(x.shape) == 4:
            a,b,c,d = x.shape
            x = super().forward(x.view(a,-1,b))
            x = x.reshape(a,b,c,d)
            return x
        if len(x.shape) == 3:
            a,b,c = x.shape
            x = super().forward(x.view(a,-1,b))
            x = x.reshape(a,b,c)
            return x

def normalization(channels):
    """
    Make a standard normalization layer.

    :param channels: number of input channels.
    :return: an nn.Module for normalization.
    """
    # return GroupNorm32(32, channels)
    return LayerNorm2D(channels)

```

Then, in the network init, I have a bunch of the following layers that do not update. 

```
self.out = nn.Sequential(
            normalization(ch),
            nn.SiLU(),
            zero_module(conv_nd(dims, input_ch, out_channels, 3, padding=1)),
        )
```

Also the model trains without any errors on Opacus and `ModuleValidator.is_valid(model)` prints True.

**Comment by woodyx218** [comment]
Hi, in the current version, the layers are supported if their classes are "torch.nn..." (see Line 791 of `supported_differentially_private_layers.py`). In your case, the normalization layers are wrapped so the class changes from "nn.GroupNorm" to "GroupNorm2D", which are not recognized. A walk-around is to modify that line.
